packages/app/src/firebaseServerApp.ts

```diff
@@ -67,6 +69,25 @@ export class FirebaseServerAppImpl
       ...serverConfig
     };
 
+    // Parse the installationAuthToken if provided.
```

Comment:
Add TODO for token verification, we shouldn't blindly trust the JWT, even if endpoints will double check

Reply:
Done.

Reply:
Done.